internal/command/init.go

```diff
@@ -31,6 +33,21 @@ import (
 	"github.com/score-spec/score-compose/internal/provisioners/loader"
 )
 
+func GetStdinFile(ctx context.Context) ([]byte, error) {
```

Comment:
Could we please move this functionality over to https://github.com/score-spec/score-go/blob/main/uriget/uriget.go#L155 as part of the file type getter? This is a good place to add unit tests too.

This is common functionality that we can use and document in all the consumers of this library. This would allow the uri to be documented as either: `file://-` or just `-`.

I can help get this turned around quickly :)